docs/articles/verifying-signatures.md

```diff
@@ -8,89 +38,87 @@ To verify the validity of a signature for an image, zot makes use of two types o
+    curl --data-binary @certificate.crt -X POST "http://localhost:8080/v2/_zot/ext/notation?truststoreType=ca"
     ```
+    *Result*
 
-The value of `resource` must be `signatures`. In addition to the requested files, the user must also specify the `tool`, which must be one of the following:
-    
-   - `cosign` for uploading public keys
-   - `notation` for uploading certificates
-
-If the uploaded file is a certificate, the user should specify these additional attributes describing the truststore:
-
-   - `truststoreType`: If the truststore is a certificate authority, the values is `ca`. This is the default if the attribute is omitted.
-
-   - `truststoreName`: The name of the truststore.
+    The uploaded file is stored in the  `_notation/truststore/x509/{truststoreType}/default` directory under the `rootDir` specified in the zot configuration file or in the Secrets Manager. The `truststores` field in the  `_notation/trustpolicy.json` file is updated automatically.
 
 ## Where needed files are stored
 
  Uploaded public keys and certificates are currently stored only in the local filesystem, in specific directories named `_cosign` and `_notation` under `$rootDir`.
 
-   - The `_cosign` directory contains uploaded public key files in the following structure:
+- The `_cosign` directory contains uploaded public key files in the following structure:
 
     ```shell
     _cosign
     ├── $publicKey1
     └── $publicKey2
     ```
 
-   - The `_notation` directory contains a set of files in the following structure:
+- The `_notation` directory contains a set of files in the following structure:
 
     ```shell
     _notation
     ├── trustpolicy.json
 	└── truststore
 	    └── x509
 	        └── $truststoreType
-	            └── $truststoreName
+	            └── default
 	                └── $certificate
     ```
 
     In this directory, the `trustpolicy.json` file contains content that is updated automatically whenever a new certificate is added to a new truststore. This content cannot be changed by the user. An example of the `trustpolicy.json` file content is shown below:
 
     ```json
     {
-        "version": "1.0",
-        "trustPolicies": [
-            {
-                "name": "default-config",
-                "registryScopes": [ "*" ],
-                "signatureVerification": {
-                    "level" : "strict" 
-                },
-                "trustStores": [],
-                "trustedIdentities": [
-                    "*"
-                ]
-            }
-        ]
-	}
+     "version": "1.0",
+      "trustPolicies": [
+        {
+          "name": "default-config",
+          "registryScopes": [ "*" ],
+          "signatureVerification": {
+            "level" : "strict" 
+          },
+          "trustStores": [],
+          "trustedIdentities": [
+            "*"
+          ]
+        }
+      ]
+    }
```

Comment:
Now the "trustpolicy.json" file contains this default content: https://github.com/project-zot/zot/blob/main/pkg/extensions/README_imagetrust.md?plain=1#L142-L161. So, it contains 2 default truststores "ca:default" and "signingAuthority:default". Also, this list of truststores will not be updated when a new certificate is uploaded.

Reply:
Added default truststores.